klog.go

```diff
@@ -1022,6 +1024,9 @@ const bufferSize = 256 * 1024
 // createFiles creates all the log files for severity from sev down to infoLog.
 // l.mu is held.
 func (l *loggingT) createFiles(sev severity.Severity) error {
+	if l.flushD != nil {
```

Comment:
under what conditions would this be nil?

Reply:
I guess it shouldn't happen during normal operation, I will remove the check.